### PR TITLE
.coafile: Add section for YML

### PR DESCRIPTION
**Pull request by mykromo**
New section for YML files only.

Fixes https://github.com/coala/coala/issues/4372

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by Monal5031** [comment]
@mykromo Please don't close PR's from next time. If you are stuck ask for help on gitter someone will help you with those git commands.
IMO the commit message doesn't look good or giving the full hint of the fix. It should be something like
```
.coafile: Add YAMLLintBear for checking yml files

Add YAMLLintBear in .coafile and fix
possible errors raised by coala locally in
the yml files.

Closes ...
```

**Comment by newbazz** [comment]
I think that the changes are correctly made!

**Comment by newbazz** [comment]
ack a4731195a44da455163cdcde9b1ac9eb77e185db

**Comment by newbazz** [comment]
@mykromo make changes in commit msg as asked by @mykromo

**Comment by Makman2** [comment]
unack a473119

**Comment by Makman2** [comment]
Btw, I think the commit shortlog is fine, if you like you can improve the body like suggested. But `Add section for YML` is quite clear for me :+1:

**Comment by nealkarpe** [comment]
@mykromo Also, please select the boxes in the checklist when you're making a pull request! :smiley:

**Comment by jayvdb** [comment]
Note this is a continuation from https://github.com/coala/coala/pull/4806 .  @mykromo deleted their fork

**Comment by newbazz** [comment]
@mykromo  take some time make all the required changes and then make a pr.

File: .misc/appveyor.yml [hunk 1]
```diff
@@ -27,15 +27,22 @@ install:
   # Prepend newly installed Python to the PATH of this build (this cannot be
   # done from inside the powershell script as it would require to restart
   # the parent CMD process).
-  - set PATH=C:\LLVM\bin;%PYTHON%;%PYTHON%\Scripts;%PATH%;C:\Program Files\Git\mingw64\bin
+  - >
+    "set PATH=C:\LLVM\bin;%PYTHON%;%PYTHON%\Scripts;%PATH%;
+    C:\Program Files\Git\mingw64\bin"
```
Comment (yashovardhanagrawal) [inline .misc/appveyor.yml]: this

**Comment by mykromo** [comment]
Let me try again. Thanks!

File: .misc/appveyor.yml [hunk 2]
```diff
@@ -27,15 +27,22 @@ install:
...
+  - >
+    "%CMD_IN_ENV% pip install --cache-dir=C:\\pip_cache -r requirements.txt
+    -r test-requirements.txt"
 
-  - curl -fsSL http://llvm.org/releases/%LLVM_VERSION%/LLVM-%LLVM_VERSION%-win32.exe -o LLVM.exe
+  - >
+    curl -fsSL
+    http://llvm.org/releases/%LLVM_VERSION%/LLVM-%LLVM_VERSION%-win32.exe
+    -o LLVM.exe
   - 7z x LLVM.exe -oC:\LLVM
```
Comment (yashovardhanagrawal) [inline .misc/appveyor.yml]: Remove these commits

File: .rultor.yml [hunk 3]
```diff
@@ -8,7 +8,9 @@ docker:
 merge:
   fast-forward: only
   script:
-    - python3 .misc/adjust_version_number.py coalib/VERSION -b `date --utc +%Y%m%d%H%M%S`
+    - >
+      python3 .misc/adjust_version_number.py
+      coalib/VERSION -b `date --utc +%Y%m%d%H%M%S`
     - bash .misc/check_maintainership.sh
```
Comment (yashovardhanagrawal) [inline .rultor.yml]: Remove these too as all these do not belong to the PR & Issue

your commits should only be present in YML file

**Comment by yashovardhanagrawal** [comment]
I think you have updated your branch but you should know the correct procedure for that
refer: https://gist.github.com/CristinaSolana/1885435

File: .rultor.yml (hunk 3 above)
Comment (Monal5031) [reply .rultor.yml]: @yashovardhanagrawal These should be here, please read the issue description. The issue needs to fix any errors raised by coala thats why earlier I pointed out to amend the commit message.

**Comment by mykromo** [comment]
I created the same branch name. How can we reopen this?

**Comment by jayvdb** [comment]
Apparently not.  Stop closing PRs and deleting repositories.

File: .rultor.yml (hunk 3 above)
Comment (yashovardhanagrawal) [reply .rultor.yml]: oh sorry, my fault
I should have realised that

anyway he has to issue a new PR now
